### PR TITLE
[jsk_perception] Add image_to_label.py

### PR DESCRIPTION
**Pull request by wkentaro**
Added:
  - jsk_perception/node_scripts/image_to_label.py

### CODE REVIEW

File: jsk_perception/test/image_to_label.test [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    </rosparam>
+  </node>
+
+  <node name="image_to_label"
+        pkg="jsk_perception" type="image_to_label.py">
+    <remap from="~input" to="image_publisher/output" />
+  </node>
+
+  <test test-name="test_image_to_label"
+        name="test_image_to_label"
```
Comment (garaemon) [inline jsk_perception/test/image_to_label.test]: Increase time-limit

Comment (wkentaro) [inline jsk_perception/test/image_to_label.test]: default is 60 secs. Is it enough, isn't it?

Comment (garaemon) [inline jsk_perception/test/image_to_label.test]: Not enough. I saw timeout error many times. I increased other tests to 360 secs

**Comment by wkentaro** [comment]
updated

**Pull request merged**